recipe/meta.yaml

```
@@ -11,10 +11,10 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
   run_exports:
-    - {{ pin_subpackage('fmt', max_pin='x') }}
+    - {{ pin_subpackage('fmt', max_pin='x.x.x') }}
```

Minor should be enough
```suggestion
    - {{ pin_subpackage('fmt', max_pin='x.x') }}
```